cranelift/codegen/src/verifier/mod.rs

```diff
@@ -1413,8 +1414,19 @@ impl<'a> Verifier<'a> {
                     BlockCallTargetType::ExNormalRet,
                     errors,
                 )?;
-                for (_tag, block) in exdata.catches() {
-                    self.typecheck_block_call(inst, block, BlockCallTargetType::Exception, errors)?;
+                for item in exdata.catches() {
+                    match item {
+                        ExceptionTableItem::Tag(_, block_call)
+                        | ExceptionTableItem::Default(block_call) => {
+                            self.typecheck_block_call(
+                                inst,
+                                &block_call,
+                                BlockCallTargetType::Exception,
+                                errors,
+                            )?;
+                        }
+                        ExceptionTableItem::Context(_) => {}
```

Comment:
Double checking my understanding: we don't need to check basic properties of the context value here (like its definition dominates this use) because these checks are handled via the generic methods of getting all uses and defs of a particular instruction elsewhere in the verifier, right?

Reply:
Yep, exactly -- the context values are included in the instruction's values iterator (`DataFlowGraph::inst_values()`) now, so are verified to exist and be valid SSA uses that way.